public/images/mozilla.svg

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
There seems to be a problem with this image... check this out https://github.com/ScottDowne/donate.mozilla.org/blob/footer-pos/public/images/mozilla.svg